src/fwd_analyzer.cpp

```
@@ -165,11 +161,14 @@ class InterleavedFwdFixpointIterator final {
     static AnalysisResult run(const Program& prog, const AnalysisContext& context, EbpfDomain entry_inv);
 };
 
-// Back-compat shims for callers that don't construct an AnalysisContext.
-AnalysisResult analyze(const Program& prog) { return analyze(prog, thread_local_analysis_context()); }
+AnalysisResult analyze(const Program& prog) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)